dts/arm/st/g0/stm32g0b1.dtsi

```diff
@@ -120,6 +120,19 @@
 		};
 	};
 
+	die_temp: dietemp {
```

Comment:
this could be a node in the stm32g0 DTS

Reply:
> STM32G0B1 datasheet: <img alt="image" width="700" src="https://user-images.githubusercontent.com/1563974/159425444-e711cd11-8b7e-40f0-bd43-cc1c7b6a2275.png">
> 
> I think the configurations should be similar across G0 family, but the TS_CAL2 seems to be missing from other series (at least G0B0 & G070), so I decided to leave them out from this PR for now.

Hmm, you sure?

Reply:
ok, that's all surprising as the refMan of the stm32g0x0 explicitly mentions TS_CAL1 and TS_CAL2 where _TS_CAL2 is the temperature sensor calibration value acquired at TS_CAL2_TEMP
(refer to the datasheet for TS_CAL2 value)_  that does not exist

Reply:
according to https://community.st.com/s/question/0D53W00000LFDS0SAP/stm32g070-how-to-calculate-the-current-ambient-temperature-from-internal-temperature-sensor
the formula seems to be : 
![TS_adc_g0x0](https://user-images.githubusercontent.com/51417392/159940281-a7072497-3206-4a1e-8cc5-b35a39869ace.png)